resources/dashboard.rb

```diff
@@ -21,6 +21,7 @@
 
 property :host,           String,   default: 'localhost'
 property :port,           Integer,  default: 3000
+property :url_path_prefix, [String, nil], default: nil
```

Comment:
Can you please address this warning (and do the same with the other warnings?)

Reply:
@ramereth could you please be more specific

Reply:
``Properties have a nil value by default so there is no need to set the default value to nil.`` This should probably just be:
```suggestion
property :url_path_prefix, String
```